projects/packages/my-jetpack/src/class-rest-product-data.php

```diff
@@ -47,67 +47,6 @@ public static function permissions_callback() {
 		return current_user_can( 'manage_options' );
 	}
 
-	/**
-	 * Gets the product data for all products
-	 *
-	 * @return array|WP_Error
-	 */
-	public static function get_all_product_data() {
-		$site_id        = \Jetpack_Options::get_option( 'id' );
-		$wpcom_endpoint = sprintf( 'sites/%d/jetpack-product-data?locale=%2$s&force=wpcom', $site_id, get_user_locale() );
```

Comment:
I see you have a separate task to remove some more stuff on the backend. Will that be possible? With the old versions of the plugin calling this API endpoint still I think we will still need to return the backupable items at least, and that's essentially all it's doing right now